chapters/renderpass.txt

```diff
@@ -1871,15 +1871,15 @@ ifdef::VK_NVX_multiview_per_view_attributes[]
     ename:VK_SUBPASS_DESCRIPTION_PER_VIEW_POSITION_X_ONLY_BIT_NVX, it must:
     also include ename:VK_SUBPASS_DESCRIPTION_PER_VIEW_ATTRIBUTES_BIT_NVX.
 endif::VK_NVX_multiview_per_view_attributes[]
-  * [[VUID-VkSubpassDescription2KHR-aspectMask-03175]]
-    The pname:aspectMask member of any element of pname:pInputAttachments
-    must: be a valid combination of elink:VkImageAspectFlagBits
-  * [[VUID-VkSubpassDescription2KHR-aspectMask-03176]]
-    The pname:aspectMask member of any element of pname:pInputAttachments
-    must: not be `0`
-  * [[VUID-VkSubpassDescription2KHR-aspectMask-02529]]
-    The pname:aspectMask member of each element of pname:pInputAttachments
-    must: not include ename:VK_IMAGE_ASPECT_METADATA_BIT
+  * If pname:attachment member of any element of pname:pInputAttachments is
+    not ename:VK_ATTACHMENT_UNUSED, then pname:aspectMask member must: be a
+    valid combination of elink:VkImageAspectFlagBits
+  * If pname:attachment member of any element of pname:pInputAttachments is
+    not ename:VK_ATTACHMENT_UNUSED, then pname:aspectMask member must: not
+    be `0`
+  * If pname:attachment member of any element of pname:pInputAttachments is
+    not ename:VK_ATTACHMENT_UNUSED, then pname:aspectMask member must: not
```

Comment:
grammatical nit: should say something like "then _its_ pname:aspectMask member" or "then _that element's_ pname:aspectMask member"

Reply:
(Same for all three)

Reply:
I kinda hate "its" in specification. Somehow it felt clearer to me without it.
With "it" it is not clear if it refers to member of pname:attachment or member of element of pname:pInputAttachments. With it ommited it feels like we talk about members of the same entity in both instances in the sentence.

Reply:
"the" is fine by me.